x-pack/plugin/eql/qa/common/build.gradle

```diff
@@ -5,6 +5,7 @@ dependencies {
   compile project(':test:framework')
   compile project(path: xpackModule('core'), configuration: 'default')
   compile project(path: xpackModule('core'), configuration: 'testArtifacts')
+  compile project(path: xpackModule('ql'), configuration: 'default')
```

Comment:
Since this import is just for `CollectionUtils`, I'd rather keep the projects separated and simply concatenate the lists through `addAll`.

Reply:
Good point. Removed.